invenio_circulation/utils.py

```diff
@@ -60,8 +60,8 @@ def is_item_available(item_pid):
     )
 
 
-def can_be_requested(item_pid=None, document_pid=None):
-    """Return if record can be requested."""
+def can_be_requested(loan):
```

Comment:
This might be tricky, take into consideration in the overlay that the loan might have both item and document and then one of them can be requested the other one not etc.. it will required a bit of thinking on how to handle this one on your side.

Reply:
We suggested introducing a type of request, but it was refused. We have implemented the logic of the can_be_requested function in our software.